docs/91_version_history.rst

```diff
@@ -35,229 +35,186 @@ Release Date: 2023-10-17
+       - An erroneous message was being produced when starting the system using the SQLite in memory storage plugin. This has now been resolved.
+       - Support has been improved for switching between different storage plugins that allows for correct schema creation when using different sqlite plugin variants for configuration and readings storage.
+       - An update has been done to the default SQLite storage plugin to enable it to handle a large number of distinct asset codes in the readings. Previously the plugin was limited in the number of assets it could support. When the number of asset codes gets large the performance of the plugin will be reduced slightly, however it will continue to ingest data.
+       - An issue that required the north service to be restarted if the source of data to send was changed in a running service has been resolved. Changing the data source no longer requires a restart of the north service.
+       - An issue that could sometimes cause a running north service to fail if the configuration for that service is updated has been resolved.
+       - A problem that prevents an updated service from restarting after an upgrade if HTTPS is used for the interface between services has been resolved.
        - A problem with incorrect timestamps in the readings graphs when the timezone of the machine was something other than UTC has been resolved.
+       - A number of inconsistencies in the timezone returned by API calls have been resolved. All API calls should return timestamps in the UTC timezone unless a timezones explicitly included in the response.
        - An issue with the code update mechanism that could cause multiple updates to occur has been resolved. Only a single update should be executed and then the flag allowing for updates to be applied should be removed. This prevents the update mechanism triggering on each restart of the system.
        - A problem that prevented the modbusC south plugin from being updated in the same way as other plugins has been resolved.
-       - The command line interface to view the status of the system has been updated to correctly show the statistics history collection task when it is running.
-       - When switch a south plugin from a slow poll rate to a faster one the new poll rate does not take effect until the en dot the current poll cycle. This could be a very long time, this has now been changed such that the south service will take the new poll rate as soon as possible rather than wait for the endow the current poll cycle.
+       - When switching a south plugin from a slow poll rate to a faster one the new poll rate does not take effect until the end of the current poll cycle. This could be a very long time, this has now been changed such that the south service will take the new poll rate as soon as possible rather than wait for the end of the current poll cycle.
```

Comment:
The sentence "This could be a very long time, this has now been changed such that the south service will take the new poll rate as soon as possible rather than wait for the end of the current poll cycle" should be split into 2 sentences at the comma. It would also be better to change "has been changed such that" to "has been changed _so_ that."